manifests/init.pp

```diff
@@ -517,6 +533,11 @@
   Optional[String] $ldap_bind_dn = $pulp::params::ldap_bind_dn,
   Optional[String] $ldap_bind_password = $pulp::params::ldap_bind_password,
   String $ldap_remote_user_attribute = $pulp::params::ldap_remote_user_attribute,
+  Boolean $enable_passthrough_pulp = $pulp::params::enable_passthrough_pulp,
+  Stdlib::Port $passthrough_pulp_http_port = $pulp::params::passthrough_pulp_http_port,
+  Optional[String] $passthrough_pulp_allowed_net = $pulp::params::passthrough_pulp_allowed_net,
```

Comment:
You can enforce the data type with https://github.com/puppetlabs/puppetlabs-stdlib/blob/master/REFERENCE.md#stdlibipaddressv4cidr. You may want to consider https://github.com/puppetlabs/puppetlabs-stdlib/blob/master/REFERENCE.md#stdlibipaddressv6cidr as well.

Reply:
@ekohl See https://github.com/theforeman/puppet-pulp/pull/372#issuecomment-499354560 regarding your answer about "full pulp instance deployment".